packages/design-system/src/components/Pagination/Page.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+export interface PageProps {
+  /** 
+   * Sets a custom url for links. Optional. 
+   */
+  customUrl?: string,
+  /**
+   * Defines the page number.
+   */
+  index: number,
+  /**
+   * Renders current page if true, other links if false.
+   */
+  isActive: boolean,
+  /** 
+   * A callback function used to handle state changes.
+   */
+  onPageChange?: (evt: React.MouseEvent) => void,
+}
+
+export default function Page({ customUrl, index, isActive, onPageChange }: PageProps): React.ReactElement {
+  return (
+    <li>
+      { isActive ? (
+        <span
+          className="ds-c-button ds-c-button--transparent ds-c-pagination__current-page"
+          aria-label={`current page, page ${index}`}
```

Comment:
I'm wondering why have the `aria-label` in addition to the `aria-current`? They're both doing the same job. Although, in my screen reader testing, they don't seem to conflict.

Reply:
I think the ARIA label was added to make the readout nicer ("page something" instead of just "number of number"). But I do wonder if a `<span>` is the correct markup here? Should the current page be able to receive focus on tab or is that unnecessary? There's also a very likely chance I'm just not using VoiceOver correctly here.

Reply:
I tested this a bit more, and realized that this part isn't getting announced in most browser/screen reader combinations we test in: `aria-label={`on page ${currentPage} of ${totalPages}`}`

Axe also [throws an error here](https://dequeuniversity.com/rules/axe/4.3/aria-allowed-attr). It's related to using `aria-label` on an element without a `role`. For more on this, [check out this post](https://www.davidmacd.com/blog/does-aria-label-override-static-text.html).

It's announced in Safari/VoiceOver, but not in Edge/Jaws, Chrome/Jaws or Firefox/NVDA. If we want to keep this copy for screen readers, I think a better approach would be to nest a span with some hidden text:

```<span className="ds-u-visibility--screen-reader">On </span>```